### DISCUSSION TITLE
Manage permission of custom filters

### DISCUSSION THREAD

**Discussion by cyclonstep** [Snowflake-Labs/terraform-provider-snowflake#2798] [Q&A] [answered]
Hello,
lately I've been wanting to grant other roles to be able to use and manage custom filters in snowflake (https://docs.snowflake.com/en/user-guide/ui-snowsight-filters#label-snowsight-custom-filter-perms) but looks like I cannot find the way to do it. Tried to search the documentation of managing custom filters of snowflake but to no avail. I thought the functions would be sit somewhere around snowflake_grant_privileges_to_role or snowflake_grant_privileges_to_account_role but am i looking at the wrong resource? 
I can actually solved this by manually enabling it in snowsight but I want to ensure that every changes in snowflake are being documented in my terraform code. 

Thank you very much!

**Comment by sfc-gh-jcieslak** [comment, marked as answer]
Hey 👋 
You can think of the provider as an abstraction over Snowflake SQL. If there are no available SQL statements able to manage the customer filters, then we cannot implement such resource because there's nothing to build on. I also checked the documentation and saw nothing regarding filters, so unfortunately for now you won't be able to manage them from the provider level. It seems the only way to interact with them is through snowsight.

**Reply by cyclonstep** [reply]
I see, so the settings need to be SQL based for terraform to be able to  apply it... Didn't know this! Learnt much from this! Thank you very much for your answer! 👍
